modules/ding_place2book/ding_place2book.module

```diff
@@ -474,57 +553,53 @@ function ding_place2book_get_event_state($event_maker_id, $event_id) {
         // we consider the sales status to be closed. In these cases there might
         // be a waiting list so we look for that before setting status closed.
         if (empty($status)) {
-          $status = !empty($meta_data->waiting_list) ? 'waiting-list' : 'closed';
+          $status = !empty($event->waiting_list) ? 'waiting-list' : 'closed';
```

Comment:
Can waiting list be active, even though status has already been set? I think yes.

Reply:
Well, it's pretty hard to test since it seems like it's not implemented in the API yet.

But I can see what you mean and I actually think that we shouldn't even look for a waiting list when ticket sales are closed, but only when open or upcoming and there are no available tickets.

I have updated the code to reflect this. Tbh I think we should just go with this, until the logic can be properly tested when the API is updated.